api/v1alpha3/machineset_webhook.go

```diff
@@ -78,3 +79,24 @@ func (m *MachineSet) validate() error {
 
 	return apierrors.NewInvalid(GroupVersion.WithKind("MachineSet").GroupKind(), m.Name, allErrs)
 }
+
+// PopulateDefaultsMachineSet fills in default field values.
+// This is also called during MachineSet sync.
+func PopulateDefaultsMachineSet(ms *MachineSet) {
```

Comment:
I think you need to add a `// +kubebuilder` annotation to generate the webhook funcs.

Reply:
gotcha. ty